spec/src/main/asciidoc/core/injectionandresolution.asciidoc

```diff
@@ -649,6 +649,9 @@ PaymentProcessor pp = anyPaymentProcessor.select(qualifier).get().process(paymen
 
 In this example, the returned bean has qualifier `@Synchronous` or `@Asynchronous` depending upon the value of `synchronously`.
 
+Qualifiers specified at `Instance` injection point or via via the `Instance.select()` method are used to perform typesafe resolution.
+If no qualifier is specified at `Instance` injection point or via the `Instance.select()` method the `@Default` qualifier is assumed.
+
```

Comment:
See also _"If no qualifier is passed to the Instance.select() method the @Default qualifier is assumed."_ sentence below.